diff --git a/node_modules/@metamask/detect-provider/dist/index.d.ts b/node_modules/@metamask/detect-provider/dist/index.d.ts
index 59d15f3..a1208c6 100644
--- a/node_modules/@metamask/detect-provider/dist/index.d.ts
+++ b/node_modules/@metamask/detect-provider/dist/index.d.ts
@@ -1,9 +1,8 @@
-interface EthereumProvider {
-    isMetaMask?: boolean;
-}
+import type { MetaMaskInpageProvider } from '@metamask/providers';
+
 declare global {
     interface Window {
-        ethereum?: EthereumProvider;
+        ethereum?: MetaMaskInpageProvider;
     }
 }
 export = detectEthereumProvider;
